packages/material/scss/_variables.scss

```diff
@@ -1289,6 +1289,42 @@ $adaptive-scheduler-current-text: $accent;
 $adaptive-scheduler-base-text: inherit;
 
 
+// Drawer
+$drawer-text: $base-text !default;
+$drawer-bg: $base-bg !default;
+$drawer-border: $base-border !default;
+$drawer-border-width: 1px !default;
+$drawer-content-padding-x: $padding-x !default;
+$drawer-content-padding-y: $padding-y !default;
+
+$drawer-scrollbar-width: 7px !default;
+$drawer-scrollbar-color: rgba(156, 156, 156, 0.7) !default;
+$drawer-scrollbar-bg: #dedede !default;
+$drawer-scrollbar-radius: 20px !default;
+$drawer-scrollbar-hovered-color: rgba(156, 156, 156, 1) !default;
+
+$drawer-item-padding-x: $padding-x !default;
+$drawer-item-padding-y: 12px !default;
+$drawer-item-font-size: 16px !default;
+$drawer-item-line-height: $line-height-lg !default;
+$drawer-icon-size: 18px !default;
+
+$drawer-initial-width: 280px !default;
+$drawer-mini-initial-width: calc(2 * #{$drawer-item-padding-x} + #{$drawer-icon-size}) !default;
+
+$drawer-hovered-text: $hovered-text !default;
+$drawer-hovered-bg: $hovered-bg !default;
+
+$drawer-focused-bg: $hovered-bg !default;
+$drawer-focused-shadow: none !default;
+
+$drawer-selected-text: $secondary !default;
+$drawer-selected-bg: $base-bg !default;
+
```

Comment:
For consistency, remove this empty row or add it in the other `_variables.scss` files